feat(helm): only keep last digits for ipv6 ports

clementnuss opened the pull request:
permits specifying ports such as :::18083, a somewhat strange but needed
syntax when trying to make emqx listen on IPv6 addresses
The solution here is to split by colon `:` and to only keep the last
entry, which will corespond to the actual number for the port.

can be tested by using following values.yaml config:
```yaml
## EMQX configuration item, see the documentation (https://hub.docker.com/r/emqx/emqx)
emqxConfig:
  EMQX_CLUSTER__DISCOVERY_STRATEGY: "dns"
  EMQX_DASHBOARD__DEFAULT_USERNAME: "admin"
  EMQX_DASHBOARD__DEFAULT_PASSWORD: "public"
  EMQX_LISTENERS__TCP__DEFAULT__BIND: ":::1234"
  EMQX_LISTENERS__TCP__INTERNAL__BIND: ":::3456"
  EMQX_DASHBOARD__LISTENERS__HTTP__BIND: ":::31415"
  EMQX_DASHBOARD__LISTENERS__HTTPS__BIND: ":::65432

```

Rory-Z commented:
I think the value of the `EMQX_LISTENERS__TCP__DEFAULT__BIND ` need a string, not list, so maybe it's will not work.
@id Am I right ?

clementnuss commented:
you are right that it is a string, but what `splitList` actually does is: split the input string, return the splitted parts as a list.

```
{{ splitList ":" ( .Values.emqxConfig.EMQX_LISTENERS__TCP__DEFAULT__BIND | default "1883" ) | last }}
``` 

so I split by column `:` and take the last entry, which will the last port number.
In the config for emqx however, the port is specified correctly with the (strange) triple-column notation, and IPv6 addresses work.

<img width="817" alt="image" src="https://github.com/emqx/emqx/assets/2116997/c4010c32-66b7-4914-b29e-0f4c53aa43b9">

Rory-Z commented:
Great, LGTM

clementnuss commented:
just did a rebase from master 🙃
